cradle-core/src/main/java/com/exactpro/cradle/PageLoader.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exactpro.cradle;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+@FunctionalInterface
+public interface PageLoader {
```

Comment:
When I look at this interface I cannot understand what it is for. How can you load a page just by `BookId`? I need to look in another place to understand that it is used to loader the first/last page of the book.
And I can't even think of a way to make this clearer except adding javadoc saying that this is used for ...
Maybe we don't need this interface at all... The built-in `Function<BookId, PageInfo>` will perfectly work I think